app/services/list_volunteer_opportunities.rb

```diff
@@ -0,0 +1,59 @@
+class ListVolunteerOpportunities
+
+  def self.with(listener, organisations, feature = Feature, http_party = HTTParty)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)